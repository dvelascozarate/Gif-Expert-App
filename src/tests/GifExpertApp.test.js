import { shallow } from 'enzyme';
import React from 'react';
import { GifExpertApp } from '../GifExpertApp';

describe('Pruebas para el componente GifExpertApp', () => {

    test('Debe mostrar el componente correctamente', () => {

        const wrapper = shallow(<GifExpertApp />);

        expect(wrapper).toMatchSnapshot();

    });

    test('Debe mostrar una lista de categorÃ­as', () => {

        const defaultCategories = ['Hulk', 'Spiderman'];

        const wrapper = shallow(<GifExpertApp defaultCategories={defaultCategories} />);

        expect(wrapper.find('GifGrid').length).toBe(defaultCategories.length);

    })

})
