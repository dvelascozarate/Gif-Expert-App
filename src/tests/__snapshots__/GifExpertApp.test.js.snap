// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas para el componente GifExpertApp Debe mostrar el componente correctamente 1`] = `
<Fragment>
  <h2>
    GifExpertApp
  </h2>
  <hr />
  <AddCategory
    setCategories={[Function]}
  />
  <ul />
</Fragment>
`;
